iceoryx_examples/singleprocess/single_process.cpp

```diff
@@ -115,6 +115,9 @@ int main()
     // set the log level to error to see the essence of the example
     iox::log::LogManager::GetLogManager().SetDefaultLogLevel(iox::log::LogLevel::kError);
 
+    /// @todo iox-#861: we need a default RouDi ID. Which one to use
```

Comment:
Thats easy we use the digit sum of the sum of the ASCII codes of the sentence `all glory to the hypnotoad`. But I can be also convinced to use the product of the ASCII codes of that sentence.

Reply:
It must be more sophisticated. We also need to take the position of the moon and other celestial objects into account.